percona/runtime/runtime.go

```diff
@@ -19,13 +19,20 @@ var refreshInterval = 60 * time.Minute
 
 // CreateRuntimeManager does the same thing as `internal/controller/runtime.CreateRuntimeManager`,
 // excet it configures the manager to watch multiple namespaces.
-func CreateRuntimeManager(namespaces string, config *rest.Config, disableMetrics bool, features feature.MutableGate) (manager.Manager, error) {
+func CreateRuntimeManager(namespaces string, config *rest.Config, disableMetrics, disableLeaderElection bool, features feature.MutableGate) (manager.Manager, error) {
 
 	options := manager.Options{
 		Cache: cache.Options{
 			SyncPeriod: &refreshInterval,
 		},
-		Scheme: r.Scheme,
+		Scheme:           r.Scheme,
+		LeaderElection:   disableLeaderElection,
+		LeaderElectionID: "08db3feb.percona.com",
```

Comment:
Why do we use an ID like this in all of our operators instead of something readable like `postgres.percona.com`?

Reply:
I have the same question, but for now, I just wanted to have it consistent.